### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by Politecoder** [recloudstream/cloudstream#1273]
### Steps to reproduce

When you go into the settings and choose clone site, once we add a clone to a site, it works for that time but when you reload or reopen the app, the cloned site disappears and  doesn't work, and if the cloned site is set as default to load as the homepage, it shows that the provider doesn't exist, I tried for multiple providers but it's not working as expected.

### Expected behavior

The cloned site should work even if we reload or reopen the app, avoiding to clone the provider again and again when the app is opened..

### Actual behavior

The clone dissapears from the list and stops working when reloaded 

### Cloudstream version and commit hash

4.4.0 4d3ab40 August 12, 2024 2:05:42 AM 

### Android version

Android 12

### Logcat

```java
None
```

### Other details

None

### Acknowledgements

- [X] I am sure my issue is related to the app and **NOT some extension**.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to pre-release version **[Latest](https://github.com/recloudstream/cloudstream/releases)**.
- [X] I will fill out all of the requested information in this form.

**Comment by Politecoder** [comment]
Yes I have enabled others in preferred media, there is a extension named Multimovies in Hindiproviders repo, the site had changed its domain therefore I was trying to clone it, but they updated the plugin now and it's working fine, but if you try to clone any of the site it works for some time and disappears after reloading..

**Comment by Luna712** [comment]
I can not reproduce this. I have had cloned sites for months with no issues.

**Comment by Politecoder** [comment]
> > Yes I have enabled others in preferred media, there is a extension named Multimovies in Hindiproviders repo, the site had changed its domain therefore I was trying to clone it, but they updated the plugin now and it's working fine, but if you try to clone any of the site it works for some time and disappears after reloading..
> 
> I tried exactly this plugin and everything worked. I also tried with non-existent website and it worked Cannot reproduce error

Did you reopen the app with being this set as the default homepage ? If not then please try once

**Comment by Luna712** [comment]
> > > Yes I have enabled others in preferred media, there is a extension named Multimovies in Hindiproviders repo, the site had changed its domain therefore I was trying to clone it, but they updated the plugin now and it's working fine, but if you try to clone any of the site it works for some time and disappears after reloading..
> > 
> > 
> > I tried exactly this plugin and everything worked. I also tried with non-existent website and it worked Cannot reproduce error
> 
> Did you reopen the app with being this(the cloned site) set as the default homepage ? If not then please try once

I did, yes.

**Comment by Politecoder** [comment]
As you can see I cloned site by the name Dynamo but when reloaded it's not present in the list

**Comment by Politecoder** [comment]
https://github.com/user-attachments/assets/8e50151f-93ee-416f-872f-e9fc420004aa

It's already selected

**Comment by Politecoder** [comment]
Is this a issue on my side or a bug ?

**Comment by Politecoder** [comment]
[logcat_2024_08_13_23_20.txt](https://github.com/user-attachments/files/16603405/logcat_2024_08_13_23_20.txt)

**Comment by Abdoumostaa** [comment]
Please add new translation sites.

**Comment by Politecoder** [comment]
What are tanslational sites ?
